frontend/package.json

```diff
@@ -101,4 +101,4 @@
     "appname": "hac-core",
     "buildrepo": "git@github.com:RedHatInsights/hac-core-frontend-build.git"
   }
-}
+}
```

Comment:
@kdoberst I think your editor removed the newline from the end of this file. Is it ok if we add it back? Thanks!

Reply:
Otherwise, this looks good to me!